docker-compose/docker-compose.yml

```diff
@@ -12,6 +12,15 @@ services:
     alfresco:
         image: alfresco/alfresco-content-repository-community:6.2.0-ga
         mem_limit: 1500m
+
+        # TODO ATS-707 - review legacy transform urls:
```

Comment:
Maybe we can remove legacy once REPO-4710 is complete and merged (see https://github.com/Alfresco/acs-community-deployment/tree/feature/repo-4710_remove_legacy_transforms).